satpy/avhrr_l1b_gaclac.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<p>Should be mention that some noaa 16 tles are provided on github?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good idea, added a link!</p><p> </p>